pages/community.js

```diff
@@ -0,0 +1,126 @@
+import useSWR from "swr";
+import styled from "styled-components";
+import sortByNewestTime from "@/utils/sortByNewestTime";
+
+export default function Community() {
+  const { data, error, isLoading } = useSWR("/api/messages");
+  const { mutate } = useSWR("/api/messages");
+
+  if (isLoading) {
+    return <p>Loading...</p>;
+  }
+
+  if (error) {
+    return <p>Failed to load messages</p>;
+  }
+
+  async function handleSubmit(event) {
+    event.preventDefault();
+    const formData = new FormData(event.target);
+    const dataMessage = Object.fromEntries(formData);
+    const date = new Date().toLocaleString();
+    const newMessage = { ...dataMessage, date: date };
+
+    const response = await fetch("/api/messages", {
+      method: "POST",
+      body: JSON.stringify(newMessage),
+      headers: { "Content-Type": "application/json" },
+    });
+    if (response.ok) {
+      mutate();
+    }
+    event.target.reset();
+  }
+  const sortedData = sortByNewestTime(data);
+
+  return (
+    <>
+      <CommunityWrapper>
+        {sortedData.map((message) => (
+          <MessageBubble key={message._id}>
+            <MessageAuthor>{message.name}</MessageAuthor>
+            <p>{message.message}</p>
+            <MessageTimestamp>{message.date}</MessageTimestamp>
+          </MessageBubble>
+        ))}
+      </CommunityWrapper>
+      <InputFormWrapper onSubmit={handleSubmit}>
+        <StyledInput type="text" placeholder="name" name="name" />
+        <StyledInput type="text" placeholder="your message..." name="message" />
```

Comment:
You want to connect your input elements with a label each for a more accessible form.
In case a label with a text would not visually fit your expectations, you could assign each input element an `aria-label` attribute. Those aria-labels are read by screen readers and provide users with a visual impairment some more information about an input field.